src/cli.rs

```diff
@@ -27,7 +27,11 @@ use crate::{buffer::Buffer, request_items::RequestItem};
 ///
 /// It reimplements as much as possible of HTTPie's excellent design.
 #[derive(StructOpt, Debug)]
-#[structopt(name = "xh", settings = &[AppSettings::DeriveDisplayOrder, AppSettings::UnifiedHelpMessage])]
+#[structopt(
+    name = "xh",
+    settings = &[AppSettings::DeriveDisplayOrder, AppSettings::UnifiedHelpMessage],
+    global_setting = AppSettings::ColoredHelp,
```

Comment:
Can we include `AppSettings::ColoredHelp` in the settings array instead?

Reply:
Ah, yes. If there are subcommands, `global_setting` will also apply to them. But it seems that there is not.

Reply:
Okay, let's use `settings` to be consistent for now.

Reply:
Will fix it in the next commit.